lib/components/ms-textarea/MsTextarea.vue

```diff
@@ -0,0 +1,136 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div class="input-container">
+    <span
+      id="label"
+      class="form-label"
+      v-show="label"
+    >
+      {{ $msTranslate(label) }}
+    </span>
+
+    <ion-textarea
+      class="textarea"
+      :class="{
+        input: !$props.disabled,
+      }"
+      ref="textareaRef"
+      :placeholder="$msTranslate(placeholder)"
+      :value="modelValue"
+      @ion-input="onChange($event.detail.value || '')"
+      :disabled="$props.disabled"
+      v-bind="$attrs"
+      :auto-grow="autogrow || false"
+      :minlength="minlength || DEFAULT_MINLENGTH"
```

Comment:
Is there a reason we want to set default values for these props? It seems easier to get rid of them, just `v-bind="$attrs"` and let ionic handle the behavior.